### PR TITLE
Tests for playground coverages

### PR DESCRIPTION
**Pull request by jonblower**
Simple way to test all the coverages in the Playground (no fixtures)

### CODE REVIEW

**Comment by jonblower** [comment]
Probably makes sense to approve and merge the coordinates-test branch before this one

File: test/test_data/playground/ReadMe.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+These are complete Coverage objects, copied from the CovJSON playground. They are included here to perform a "sanity check" that all the coverages validate against the schema.
```
Comment (letmaik) [inline test/test_data/playground/ReadMe.txt:1]:
```suggestion
These are complete CoverageJSON objects, copied from the CovJSON playground. They are included here to perform a "sanity check" that all the coverages validate against the schema.
```

Also has Domain and NdArray objects.

Comment (letmaik) [inline test/test_data/playground/ReadMe.txt:1]: Can you rename this to README.txt? That's a more common casing.

File: test/test_playground_coverages.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+import os
+import pytest
+import json
+
+pytestmark = pytest.mark.schema("/schemas/coveragejson")
+
+def test_all_playground_coverages(validator):
+
+    # Recursively traverse the test_data/playground directory
+    rootpath = os.path.join(os.path.dirname(__file__), "test_data/playground")
```
Comment (letmaik) [inline test/test_playground_coverages.py:12]:
```suggestion
    rootpath = os.path.join(os.path.dirname(__file__), "test_data", "playground")
```

**Review by letmaik — COMMENTED** [review]
Looks good, just some minor comments.
Re merging, you may get into trouble slightly due to squash merging the other PR, but we'll see.

**Comment by jonblower** [comment]
Superseded by PR 31

**Pull request closed without merging**